### DISCUSSION TITLE
Encore.go - Render HTML to support HTMX like stacks

### DISCUSSION THREAD

**Discussion by berkan-cetinkaya** [encoredev/encore#1421] [Suggestions]
Hello, I'm wondering if it's possible to render html instead of returning json without using a fallback approach like gin natively?

**Comment by berkan-cetinkaya** [comment]
@marcuskohlberg , any input?

**Reply by ekerfelt** [reply]
Hi @berkan-cetinkaya. Your best approach at the moment would be to use a raw endpoint: https://encore.dev/docs/ts/primitives/raw-endpoints. Using that, you'd be able to render html using a library of your choice and write the response to the response writer.

**Reply by berkan-cetinkaya** [reply]
Hello @ekerfelt , thanks for the reply

so for go https://encore.dev/docs/primitives/raw-endpoints but it would be kind of hack as encore is not designed to support htmx like stacks that's what I want to confirm

Best,
